src/lib/mpq/chain.js

```diff
@@ -0,0 +1,40 @@
+import path from 'path';
+
+import glob from 'globby';
+
+import MPQ from './';
+
+class MPQChain {
+  static PATTERNS = [
```

Comment:
Could we keep this private by moving it out of `class MPQChain` and defining it as a `const` just above?

Reply:
Ah, much better, will do 👍